tests/ubench/CMakeLists.txt

```diff
@@ -0,0 +1,33 @@
+include(ExternalProject)
+
+set(gbench_install_dir "${PROJECT_BINARY_DIR}/gbench")
+
+set(gbench_cmake_args
+    -DCMAKE_BUILD_TYPE=release
+    -DCMAKE_INSTALL_PREFIX=${gbench_install_dir}
+    -DCMAKE_C_COMPILER=${CMAKE_C_COMPILER}
+    -DCMAKE_CXX_COMPILER=${CMAKE_CXX_COMPILER})
+
+ExternalProject_Add(gbench
+    SOURCE_DIR "${CMAKE_CURRENT_SOURCE_DIR}/google-benchmark"
+    CMAKE_ARGS "${gbench_cmake_args}"
+#    INSTALL_DIR "${gbench_install_dir}"
```

Comment:
Or perhaps put back in. I'll do one or the other! `INSTALL_DIR` in `ExternalProject_Add` does almost, but not quite, nothing.